Sources/App/Controllers/PackageController+routes.swift

```diff
@@ -144,15 +144,15 @@ enum PackageController {
         let catchAll = [archive].compactMap { $0 } + req.parameters.getCatchall()
         let path: String
         switch fragment {
-            case .documentation, .tutorials:
-                // DocC lowercases documentation and tutorial URLs. Since these routes can also
+            case .data, .documentation, .tutorials:
+                // DocC lowercases "target" names in URLs. Since these routes can also
                 // appear in user generated content which might use uppercase spelling, we need
-                // to lowercase the input in these cases.
+                // to lowercase the input in certain cases.
                 // See https://github.com/SwiftPackageIndex/SwiftPackageIndex-Server/issues/2168
                 // and https://github.com/SwiftPackageIndex/SwiftPackageIndex-Server/issues/2172
                 // for details.
                 path = catchAll.joined(separator: "/").lowercased()
-            case .css, .data, .faviconIco, .faviconSvg, .images, .img, .index, .js, .themeSettings:
+            case .css, .faviconIco, .faviconSvg, .images, .img, .index, .js, .themeSettings:
```

Comment:
I tried making the `lowercased()` apply for all fragments but we've actually got a test in place in `test_documentation_images` that asserts:

```swift
        try app.test(.GET, "foo/bar/1.2.3/images/BAZ.png") {
            // validation
            XCTAssertEqual($0.status, .ok)
            XCTAssertEqual($0.content.contentType?.description, "application/octet-stream")
            XCTAssertEqual($0.body.asString(), "/foo/bar/1.2.3/images/BAZ.png")
        }
```

and which fails if we do that. So at the very least `.images` we'd need to leave alone.

But it's probably safer to do this bit-by-bit in case other fragments needs this rather than risk breaking more with a big change.